src/styles/mentors.scss

```diff
@@ -0,0 +1,105 @@
+.mentor-card {
+  &.card {
+    border-radius: 10px;
+    margin-bottom: 16px;
+    cursor: pointer;
+  }
+
+  .card-title {
+    font-size: 18px;
+    font-weight: bold;
+    margin-bottom: 4px;
+  }
+
+  .card-text {
+    font-size: 14px;
+    line-height: 16px;
+  }
+}
+
+.mentor-modal.show {
+  display: flex;
+}
+
+.mentor-modal {
+  .modal-dialog {
+    margin: auto;
+    max-width: unset;
+  }
+
+  .modal-content {
+    flex-direction: row;
+    padding: 32px;
+    border-radius: 24px;
+    border: none;
+    position: relative;
+  }
+
+  .mentor-img {
+    width: 350px;
+    height: auto;
+    margin-bottom: 16px;
+    border-radius: 16px;
+  }
+
+  .mentor-name {
+    font-size: 18px;
+    font-weight: bold;
+    margin-bottom: 4px;
+    margin-left: 8px;
+  }
+
+  .mentor-pos {
+    font-size: 14px;
+    line-height: 16px;
+    margin-left: 8px;
+    margin-bottom: 0;
+  }
+
+  &--heading {
```

Comment:
I know other .scss files seem to be using __ rather than --. not sure if it makes a difference but stylistically for consistency should we pick one?

Reply:
👍 i'll change mine